Presentation.Web/app/components/it-advice/it-advice-modal-view.html

```diff
@@ -50,7 +50,7 @@ <h3 class="modal-title pull-left">{{advisName}}</h3>
             <!--Afsendelsestype-->
             <div class="col-md-6">
                 <div class="form-group">
-                    <label for="advisType" class="control-label small">Afsendelsestype <span class="text-danger">*</span></label>
+                    <label for="adviceType" class="control-label small">Afsendelsestype <span class="text-danger">*</span></label>
```

Comment:
obs: De to der skal matche. Du har lige ændret det så den peger på ingenting :-)
![image](https://user-images.githubusercontent.com/13116889/119127886-2f07b780-ba35-11eb-990f-8abea0517777.png)